crates/proof-of-sql/tests/integration_tests.rs

```diff
@@ -323,7 +323,7 @@ fn we_can_prove_a_complex_query_with_curve25519() {
         "sxt.table".parse().unwrap(),
         owned_table([
             smallint("a", [1_i16, 2, 3]),
-            int("b", [1_i32, 0, 1]),
+            int("b", [1_i32, 4, 3]),
             bigint("c", [3_i64, 3, -3]),
             bigint("d", [1_i64, 2, 3]),
             varchar("e", ["d", "e", "f"]),
```

Comment:
could we test boundaries here as well

Reply:
@drcapybara I mean we can but I think we don't need to. The purpose of this test is to check a bunch of random features in a single complex query.